.nxignore

```diff
@@ -1,5 +1,6 @@
 ; build output
 **/dist/**
+tmp
```

Comment:
tmp is gitignored , adding explicitly to avoid nx graph creation issues ( they should not happen though )